diff --git a/Arduino15/packages/SPRESENSE/hardware/spresense/1.1.3/libraries/Audio/Audio.cpp b/Arduino15/packages/SPRESENSE/hardware/spresense/1.1.3/libraries/Audio/Audio.cpp
index 3e48164..fbe6ee5 100644
--- a/Arduino15/packages/SPRESENSE/hardware/spresense/1.1.3/libraries/Audio/Audio.cpp
+++ b/Arduino15/packages/SPRESENSE/hardware/spresense/1.1.3/libraries/Audio/Audio.cpp
@@ -417,7 +417,9 @@ err_t AudioClass::setReadyMode(void)
       return AUDIOLIB_ECODE_AUDIOCOMMAND_ERROR;
     }
 
-  board_external_amp_mute_control(true);
+  // To minimize the pop noise as possible, the mute control is performed
+  // in the application side. Therefore, remove it from audio library.
+  //board_external_amp_mute_control(true);
 
   destroyStaticPools();
 
@@ -488,7 +490,9 @@ err_t AudioClass::setPlayerMode(uint8_t device, uint8_t sp_drv)
       return AUDIOLIB_ECODE_AUDIOCOMMAND_ERROR;
     }
 
-  board_external_amp_mute_control(false);
+  // To minimize the pop noise as possible, the mute control is performed
+  // in the application side. Therefore, remove it from audio library.
+  //board_external_amp_mute_control(false);
 
   return AUDIOLIB_ECODE_OK;
 }
