diff --git a/Arduino15/packages/SPRESENSE/hardware/spresense/1.1.3/libraries/Audio/Audio.h b/Arduino15/packages/SPRESENSE/hardware/spresense/1.1.3/libraries/Audio/Audio.h
index 2675bb0..794adc0 100644
--- a/Arduino15/packages/SPRESENSE/hardware/spresense/1.1.3/libraries/Audio/Audio.h
+++ b/Arduino15/packages/SPRESENSE/hardware/spresense/1.1.3/libraries/Audio/Audio.h
@@ -49,7 +49,7 @@
 #define WRITE_FIFO_FRAME_SIZE (1024*2*3)
 #define WRITE_BUF_SIZE   (WRITE_FIFO_FRAME_NUM * WRITE_FIFO_FRAME_SIZE)
 
-#define READ_FIFO_FRAME_NUM   (10)
+#define READ_FIFO_FRAME_NUM   (1)
 #define READ_FIFO_FRAME_SIZE  (1024*2*8)
 #define READ_BUF_SIZE    (READ_FIFO_FRAME_NUM * READ_FIFO_FRAME_SIZE)
 
